### PR TITLE
lightning: Add link to Colab notebook.

### PR DESCRIPTION
**Pull request by daavoo**
Per https://github.com/iterative/dvclive/pull/599

### CODE REVIEW

**Comment by dberenbaum** [comment]
Could you also use the snippet from the notebook to add a metric logging example directly in the docs page like we have for hyperparemeters? Maybe we should start the examples with logging metrics, then logging hyperparameters, then the others. WDYT?

**Comment by daavoo** [comment]
> Could you also use the snippet from the notebook to add a metric logging example directly in the docs page like we have for hyperparemeters? Maybe we should start the examples with logging metrics, then logging hyperparameters, then the others. WDYT?

I restructured and moved that to the `Usage` section. I think the snippet is still of a reasonable size.
PTAL

**Comment by daavoo** [comment]
ping @dberenbaum

File: content/docs/dvclive/ml-frameworks/pytorch-lightning.md [hunk 1]
```diff
@@ -5,33 +5,54 @@ DVCLive allows you to add experiment tracking capabilities to your
...
+class LitModule(pl.LightningModule):
+    def __init__(self, layer_1_dim=128, learning_rate=1e-2):
+        super().__init__()
+        # layer_1_dim and learning_rate will be logged by DVCLive
+        self.save_hyperparameters()
+
+    def training_step(self, batch, batch_idx):
+        metric = ...
+        # some_metric will be logged by DVCLive
+        self.log("some_metric", metric)
```
Comment (dberenbaum) [inline content/docs/dvclive/ml-frameworks/pytorch-lightning.md:39]:
```suggestion
        self.log("train_metric", metric, on_step=False, on_epoch=True)
```

Maybe we should include an example like this and explain how it will get logged to make it more concrete than output format section below?

Comment (daavoo) [reply content/docs/dvclive/ml-frameworks/pytorch-lightning.md:39]: I see the point of the `train_metric` but I am unsure about the other args.
Why do we want to show those details? Feels like duplicate lightning docs

Comment (dberenbaum) [reply content/docs/dvclive/ml-frameworks/pytorch-lightning.md:39]: The idea was to show how we parse it into separate `step`/`epoch` folders.

This whole section duplicates a lot of lightning docs, but I think it's more user-friendly this way.

Is your concern about maintenance? Or that it makes the example look complicated? Something else?

Comment (dberenbaum) [reply content/docs/dvclive/ml-frameworks/pytorch-lightning.md:39]: @daavoo ping

**Comment by dberenbaum** [comment]
Should close #4607

File: content/docs/dvclive/ml-frameworks/pytorch.md [hunk 2]
```diff
@@ -3,41 +3,77 @@
...
 
 ```python
-from dvclive import Live
+from DVCLive import Live
+
+...
+
+with Live(save_dvc_exp=True, report="notebook") as live:
+
+    live.log_params(params)
```
Comment (dberenbaum) [inline content/docs/dvclive/ml-frameworks/pytorch.md:35]: Not a blocker, but it would be nice to find a way to highlight the dvclive-specific code in these examples since they are long and include plenty of code that's not really about dvclive.

File: content/docs/dvclive/ml-frameworks/pytorch-lightning.md [hunk 3]
```diff
@@ -5,33 +5,54 @@ DVCLive allows you to add experiment tracking capabilities to your
...
-```py
-{Live.plots_dir}/metrics/{split}/{iter_type}/{metric}.tsv
-```
+<admon type="info">
 
-Where:
+By default, PyTorch Lightning creates a directory to store checkpoints using the
+logger's name (`DVCLiveLogger`). You can change the checkpoint path or disable
+checkpointing at all as described in the
+[PyTorch Lightning documentation](https://lightning.ai/docs/pytorch/stable/common/checkpointing_basic.html#disable-checkpointing)
```
Comment (dberenbaum) [inline content/docs/dvclive/ml-frameworks/pytorch-lightning.md:53]: We have this admon and then a snippet in the examples about how to checkpoint. Should we be more opinionated and suggest one workflow here, or at least connect between this admon and the example below somehow?

Comment (daavoo) [reply content/docs/dvclive/ml-frameworks/pytorch-lightning.md:53]: I think we could drop the admon and leave the snippet as reference for working with checkpoints

Comment (dberenbaum) [reply content/docs/dvclive/ml-frameworks/pytorch-lightning.md:53]: Let's make it a follow-up issue. Don't want to block this PR.

**Review by dberenbaum — APPROVED** [review]
